doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -0,0 +1,304 @@
+(observability-configure-manage-dashboard)=
+# Configuring and Managing Ray Dashboard
+{ref}`Ray Dashboard<observability-getting-started>` is one of the most important tools to monitor and debug Ray applications and clusters. This page describes how to configure Ray Dashboard on your clusters.
+
+Dashboard configurations may differ depending on how you launch Ray clusters (e.g., local Ray cluster v.s. KubeRay). Integrations with Prometheus and Grafana are optional for better dashboard experience.
+
+:::{note}
+At the moment, Ray Dashboard is only intended for interactive development and debugging because dashboard UI and the underlying data are not accessible any more after the clusters are terminated. For production monitoring and debugging, users should rely on [persisted logs](../cluster/kubernetes/user-guides/logging.md), [persisted metrics](./metrics.md), [persisted Ray states](../ray-observability/user-guides/cli-sdk.rst), and other observability tools.
+:::
+
+## Changing Ray Dashboard Port
+Ray Dashobard runs on port `8265` of the head node. Follow the instructions below to customize the port if needed.
+
+::::{tab-set}
+
+:::{tab-item} Single-node local cluster
+**Start the cluster explicitly with CLI** <br/>
+Pass the ``--dashboard-port`` argument with ``ray start`` in the command line.
+
+**Start the cluster implicitly with `ray.init`** <br/>
+Pass the keyword argument ``dashboard_port`` in your call to ``ray.init()``.
+:::
+
+:::{tab-item} VM Cluster Launcher
+To be added
+:::
+
+:::{tab-item} Kuberay
+View [specifying non-default ports](https://docs.ray.io/en/latest/cluster/kubernetes/user-guides/config.html#specifying-non-default-ports) page for details.
+:::
+
+::::
+
+
+## Viewing Ray Dashboard in browsers
+When you start a single-node Ray cluster on your laptop, you can access the dashboard through a URL printed when Ray is initialized (the default URL is `http://localhost:8265`).
+
+
+When you start a remote Ray cluster via the {ref}`VM cluster launcher <vm-cluster-quick-start>`, {ref}`KubeRay operator <kuberay-quickstart>` or manual configuration, Ray Dashboard will be launched on the head node but the dashboard port may not be publicly exposed. Additional setup is needed if you want to access the Ray Dashboard from outside the head node.
+
+::::{tab-set}
+
+:::{tab-item} VM Cluster Launcher
+**Port forwarding** <br/>
+You can securely port-forward local traffic to the dashboard via the ``ray
+dashboard`` command.
+
+```shell
+$ ray dashboard [-p <port, 8265 by default>] <cluster config file>
+```
+
+The dashboard is now be visible at ``http://localhost:8265``.
+:::
+
+:::{tab-item} Kuberay
+
+The KubeRay operator makes the dashboard available via a Service targeting the Ray head pod, named ``<RayCluster name>-head-svc``. You can access the
+dashboard from within the Kubernetes cluster at ``http://<RayCluster name>-head-svc:8265``.
+
+**Add Ingress rule**
+
```

Comment:
Need input from @kevin85421

Reply:
Do you mean https://github.com/ray-project/kuberay/blob/master/docs/guidance/ingress.md?

Reply:
I think so. This is to address the comment left by @DmitriGekhtman